frontend/src/components/common/button/button.tsx

```diff
@@ -26,6 +27,7 @@ const Button: FC<Props> = ({
   to,
   onClick,
   iconName,
+  iconColor,
```

Comment:
Let's provide default value if it's optional prop
```suggestion
  iconColor = 'blue',
```